deploy/helm-chart/values.yaml

```diff
@@ -0,0 +1,51 @@
+nginxGateway:
+  ## A GatewayClass resource with the name equal to the class must be deployed. Otherwise, NGINX Kubernetes Gateway
+  ## will fail to start. NGINX Kubernetes Gateway only processes resources that belong to its class - i.e. have the
+  ## "gatewayClassName" field resource equal to the class.
+  gatewayClass: "nginx"
```

Comment:
if we support multiple releases of NKG per cluster, let's document that this value needs to be unique per each release.